### PR TITLE
Credit debit screen

### PR DESCRIPTION
**Pull request by AhmedAbdallah125**
IMAGE 2022-05-20 17:53:46
the google image will be replaced by paypal which exists in other branch which will be merged after

### CODE REVIEW

File: app/src/main/res/navigation/mobile_navigation.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/mobile_navigation"
-    app:startDestination="@+id/navigation_home">
+    app:startDestination="@id/navigation_home">
```
Comment (MohabElziny) [inline app/src/main/res/navigation/mobile_navigation.xml:6]: you removed "+" after "@"

File: app/src/main/res/layout/fragment_credit.xml [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+                </com.google.android.material.textfield.TextInputLayout>
+
+                <TextView
+                    android:id="@+id/textView0"
+                    style="@style/fontBoldStyle"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_marginStart="8dp"
+                    android:layout_marginTop="8dp"
+                    android:text="@string/expire_data"
```
Comment (MohabElziny) [inline app/src/main/res/layout/fragment_credit.xml:116]: Do you use the same string for two text views?

Comment (AhmedAbdallah125) [reply app/src/main/res/layout/fragment_credit.xml:116]: yes it's same

File: app/src/main/res/navigation/mobile_navigation.xml (hunk 1 above)
Comment (AhmedAbdallah125) [reply app/src/main/res/navigation/mobile_navigation.xml:6]: done

**Pull request merged**